### ISSUE TITLE
How to protect pages?

### ISSUE THREAD

**Issue by sts-ryan-holton** [james2doyle/nuxt-firebase-auth#4]
How can I protect a page until a user is logged in?

This is my attempt at protecting a page which doesn't seem to work:
```
<template>
  <div>
    secret page
  </div>
</template>

<script>
export default {
  middleware: 'authenticated'
}
</script>
```

**Comment by james2doyle** [comment]
Check for the account in the store. See this page: https://github.com/james2doyle/nuxt-firebase-auth/blob/master/pages/account/index.vue

**Comment by sts-ryan-holton** [comment]
@james2doyle  ideally I want to be redirected away from my secret page if I'm not logged in, how can I do this?

**Comment by james2doyle** [comment]
The login page has an example of this. Have you browsed the code or run the
example?

**Comment by sts-ryan-holton** [comment]
@james2doyle  I've just reviewed the code on the login page. I'm able to automatically redirect to the login page from a secret page once logged in however this isn't the behaviour I'm looking for.

I've tried to make some adjustments to my secret page. If I'm not logged in and try to access the secret page I should be redirected somewhere else, but this isn't working for me, my code is as follows my my secret page:

```
<template>
  <div>
    secret page
  </div>
</template>

<script>
import { mapState } from 'vuex'

export default {
  middleware: 'anonymous',
  watch: {
    user (to, from) {
      this.$router.push('/account/login')
    }
  },
  computed: mapState([
    'user'
  ])
}
</script>
```

What am I missing, thanks :)
